src/sentry/utils/performance_issues/base.py

```diff
@@ -65,6 +68,7 @@ class DetectorType(Enum):
     DetectorType.SLOW_DB_QUERY: "performance.issues.slow_db_query.problem-creation",
     DetectorType.RENDER_BLOCKING_ASSET_SPAN: "performance.issues.render_blocking_assets.problem-creation",
     DetectorType.M_N_PLUS_ONE_DB: "performance.issues.m_n_plus_one_db.problem-creation",
+    DetectorType.DB_MAIN_THREAD: "performance.issues.db_main_thread.problem-creation",
```

Comment:
Goes with the previous comment, if you use the auto feature handler you can use
```suggestion
    DetectorType.DB_MAIN_THREAD: "performance.issues.db_main_thread.flag_disabled",
```
reference https://github.com/getsentry/sentry/pull/45797/files#diff-4bcc09e0376432f8c06c7232afd12c430472607c1c2b0b0e53f7bfd43705054aR62